src/lib/project.js

```diff
@@ -314,12 +315,13 @@ async function scaffoldNext(projectName) {
   // set the project name and default flags
   // https://nextjs.org/docs/api-reference/create-next-app#options
   let args = [
-    'create-next-app@14.2.3',
+    'create-next-app@14.2.12',
     'ui',
     '--use-npm',
-    '--src-dir',
+    '--no-src-dir',
+    '--ts',
     '--import-alias "@/*"',
-    '--no-app',
+    '--app',
```

Comment:
Should we add this change to the [changelog](https://github.com/o1-labs/zkapp-cli/blob/main/CHANGELOG.md)?